documentation.md

```diff
@@ -12,6 +12,7 @@ navigation:
 <p>Setup instructions, programming guides, and other documentation are available for each stable version of Spark below:</p>
 
 <ul>
+  <li><a href="{{site.baseurl}}/docs/2.2.1/">Spark 2.2.1</a></li>
```

Comment:
this was never listing the latest for some reasons
I was documented it as it was, but we could change this https://github.com/apache/spark-website/blame/asf-site/release-process.md#L241

Reply:
You are right; I figured add it for completeness rather than consistency. I think the release docs are close enough still as they indicate this needs to change and it's clear how.